### PR TITLE
New board update

### PR DESCRIPTION
**Pull request by zcardwell**
Just updating the board! Removed folks just according to my current list (happy to add folks back in!). 

Highlights:
- Made "email" optional since I haven't built them on the google admin page yet. 
- Commented out `hpricot` - couldn't get it to install and looking at the repo looks like it is deprecated. I can build with it, but as a nonrubyist, I don't know if I'm inviting programming doom down the line - just wanted to test the page changes before pushing. 
- Formatted photos and etc. 

If I missed anything just let me know!

### CODE REVIEW

**Review by ejmudrak — APPROVED** [review]
LGTM 👍  couldn't find any typos on the members added at least.

File: Gemfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ group :development, :test do
   gem "launchy"
   gem "spork"
 
-  gem "hpricot"
+  #gem "hpricot"
```
[the remark this answers is not in the document]
Comment (dummied) [reply Gemfile:42]: It's only in the dev/test environment, so removing it should be fine. Also see no other references to it in the app. Don't have time to get the app fully running right this second (perhaps later!), but removing it entirely should be fine.

Aside: that's a gem name I haven't heard in quite some time! Warms the cockles of my heart greatly to see an old friend.

**Review by armilam — APPROVED** [review]
The linter is mad about vulnerable gems, but that is something we can resolve separately. This looks like it should work. Go ahead and merge it - hopefully the auto-deploy still works 😄

File: Gemfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (azurewraith) [reply Gemfile:42]: That's what I get for code reviewing on my phone 😂

In one of my less popular opinions I believe `_why` went on to do all the art for Chipotle's everywhere 😅

File: app/views/site/about.html.erb [hunk 2]
```diff
@@ -20,48 +19,114 @@ board_members = [
...
+    slack_url: 'https://indyhackers.slack.com/team/UDTS3H8E7',
+    slack_handle: '@josh',
+    linkedin: 'https://www.linkedin.com/in/josh-hurst42/'
+  },
+  {
+    name: 'Ben Cochrane',
+    picture: 'ben.jpg',
+    #email:'ben@indyhackers.org',
+    slack_url:'https://indyhackers.slack.com/team/UER6V5CMN',
+    slack_handle: '@Ben Cochrane',
```
[the remark this answers is not in the document]
Comment (azurewraith) [reply app/views/site/about.html.erb:92]: 🖖
We named our oldest son Zephram 😁

**Pull request merged**